deliver/lib/deliver/upload_metadata.rb

```diff
@@ -18,14 +18,26 @@ class UploadMetadata
     # Make sure to call `load_from_filesystem` before calling upload
     def upload(options)
       return if options[:skip_metadata]
-      verify_available_languages!(options)
+      # it is not possible to create new languages, because
+      # :keywords is not write-able on published versions
+      # therefore skip it.
+      verify_available_languages!(options) unless options[:edit_live]
 
       app = options[:app]
 
       details = app.details
-      v = app.edit_version(platform: options[:platform])
+      if options[:edit_live]
+        # not all values are editable when using live_version
+        v = app.live_version(platform: options[:platform])
+        localised_options = [:description, :release_notes, :support_url, :marketing_url]
```

Comment:
Maybe it would be more consistent to add `LOCALISED_LIVE_VERSION_VALUES` and `NON_LOCALISED_LIVE_VERSION_VALUES`? But this is good enough.